source/numerics.tex

```diff
@@ -4009,7 +4009,7 @@
  each indexing operator applied to \tcode{begin} is to be taken modulo $n$,
  and $T(x)$ is defined as $x \, \xor \, (x \, \rightshift \, 27)$:
```

Comment:
Unrelated to the existing patch, but the extra spacing here caught my eye. The `\xor` and `\rightshift` macros already include thin spaces around them. May I recommend removing all instances of `\,` both in the text here and in the `\xor` and `\rightshift` (and similar) macros and instead using `\mathbin` to handle the spacing?

Reply:
Update: I created PR #2073 to address this issue.